### PR TITLE
ass-sembler

### PR DESCRIPTION
**Pull request by Rundas01**
assembler recipes + more variety in tiered cables/wires

### CODE REVIEW

**Review by Trinsdar — APPROVED** [review]

File: common/src/main/java/muramasa/gregtech/data/TierMaps.java [hunk 1]
```diff
@@ -107,21 +107,21 @@ public static void providerInit() {
         {
             ImmutableMap.Builder<Tier, PipeItemBlock> builder = ImmutableMap.builder();
             builder.put(Tier.ULV, WIRE_LEAD.getBlockItem(PipeSize.VTINY));
-            builder.put(Tier.LV, WIRE_TIN.getBlockItem(PipeSize.VTINY));
-            builder.put(Tier.MV, WIRE_COPPER.getBlockItem(PipeSize.VTINY));
-            builder.put(Tier.HV, WIRE_GOLD.getBlockItem(PipeSize.TINY));
-            builder.put(Tier.EV, WIRE_ANNEALED_COPPER.getBlockItem(PipeSize.SMALL));
-            builder.put(Tier.IV, WIRE_PLATINUM.getBlockItem(PipeSize.TINY));
+            builder.put(Tier.LV, WIRE_COPPER.getBlockItem(PipeSize.VTINY));
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/data/TierMaps.java]: Why was this changed to copper? Copper is tier 2, not tier 1

Comment (Rundas01) [reply common/src/main/java/muramasa/gregtech/data/TierMaps.java]: because I wanted to not use the same material for cables and wires. in that process, I accidentally used an mv wire for lv. will change it to another 32 EU wire asap

**Comment by Trinsdar** [comment]
is this ready yet?

**Comment by Rundas01** [comment]
> is this ready yet?

yes, otherwise, I would have made it a draft
